Webrew.Client/ClientApp/src/app/display-card/components/display-card/display-card.component.html

```diff
@@ -1,13 +1,25 @@
-<mat-card>
-	<mat-card-header>
-		<h3 mat-card-title>Dunkelweizen</h3>
-	</mat-card-header>
-	<div mat-card-subtitle
-		class="image-container">
-		<img class="image"
-			src="https://s3.amazonaws.com/cdn.greatfermentations.com/images/popup/bka042_dunkelweizen_extract_kit.jpg" />
-	</div>
-	<mat-card-content>
-
-	</mat-card-content>
-</mat-card>
+<mat-card class="standard-card">
+    <mat-card-header >
+      <mat-card-title>{{ beer.name }}</mat-card-title>
+      <mat-card-subtitle>{{ beer.company }}</mat-card-subtitle>
+    </mat-card-header>
+    <a href="/beer/{{beer.id}}">
+      <img class="card-image" src="{{ beer.imageURL }}" alt="Photo of {{ beer.name }}" >
+    </a>
+  <mat-card-content>
+    <hr />
+    <div mat-card-style class="float-left">Style: {{ beer.style }}</div>
+    <div mat-card-avgrating class="float-right">Avg Rating: {{ beer.avgRating }}</div>
+    <div mat-card-abv class="float-center">ABV: {{ beer.abv }}%</div>
+    <hr />
+    <div>
+      <p mat-card-desc>{{ beer.description }}</p>
```

Comment:
These attributes like "mat-card-desc" and "mat-card-abv" I'm pretty sure don't do anything unless it's an angular 8 thing

Reply:
I wasn't sure how you had named the previous ones so I had just named each after the data that was in it.